src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileContent = readDataFromFile(fromFileName);
+        int[] data = calculateData(fileContent);
+
+        writeDataToFile(toFileName, prepareReport(data));
+    }
+
+    private static String readDataFromFile(String path) throws RuntimeException {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(path))) {
+            StringBuilder stringBuilder = new StringBuilder();
+            String line = null;
+
+            while ((line = bufferedReader.readLine()) != null) {
+                stringBuilder.append(line).append(System.lineSeparator());
+            }
+
+            return stringBuilder.toString();
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot read the file", e);
```

This violates checklist item: "Don't ignore exceptions". The exception message should include the file name (path) to help identify which file caused the error.